File: programming-challenge/src/main/java/com/programming/programmingchallenge/model/Movements.java
package com.programming.programmingchallenge.model;

import javax.persistence.*;
import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Table(name = "movement")
@Entity
public class Movements {

    @Id
    private int ID;

    @Column(name = "account_company")
    private String account_company;

    @Column(name = "new_movement_reason")
    private String new_movement_reason;

    @Column(name = "new_species")
    private String new_species;

    @Column(name = "new_origin_premid")
    private String new_origin_premid;

    @Column(name = "new_destination_premid")
    private String new_destination_premid;

    @Column(name = "new_num_of_item_moved")
    private int new_num_of_item_moved;

    @Column(name = "new_shipment_start_date")
    private Date new_shipment_start_date;


}

File: pc_fe/src/components/Movements.js
import * as ReactBootStrap from 'react-bootstrap'

const Movements = (props) => {
  console.log("ansM =",  props.ans)

  return(
    <div>
      <h2> Movements Data </h2>
      <ReactBootStrap.Table striped bordered hover>
      <tbody>
        <tr>
          <th>ID</th>
          <th>Account Company</th>
          <th>Movement Reason</th>
          <th>Species</th>
          <th>Origin Premise ID</th>
          <th>Destination Premise ID</th>
          <th>Number of Items moved</th>
          <th>Shipment Start Date</th>
        </tr>
        {
          props.ans && props.ans.map((e) => (
            <tr key={e.id}>
              <td>{e.id}</td>
              <td>{e.account_company}</td>
              <td>{e.new_movement_reason}</td>
              <td>{e.new_species}</td>
              <td>{e.new_origin_premid}</td>
              <td>{e.new_destination_premid}</td>
              <td>{e.new_num_of_item_moved}</td>
              <td>{e.new_shipment_start_date}</td>
            </tr>  
          ))
        }
      </tbody>
    </ReactBootStrap.Table>
    </div>
  )
}

export default Movements